challenges/10exercises.js

```diff
@@ -1,26 +1,181 @@
-// Define a function max() that takes two numbers as arguments and returns the largest of them.
-// Use the if-then-else construct available in Javascript.
+// // 1. Define a function max() that takes two numbers as arguments and returns the largest of them.
+function max(number1, number2) {
+  if (number1 > number2) {
+    return number1;
+  }
+  if (number2 > number1) {
+    return number2;
+  }
+  if (number1 === number2) {
+    return "The two numbers are equal."
+  }
+}
+// // Use the if-then-else construct available in Javascript. ??(Wouldn't work for equal)
+//
+// // 2. Define a function maxOfThree() that takes three numbers as arguments and returns the largest of them.
+  function maxOfThree(number1, number2, number3) {
+    return Math.max(number1, number2, number3);
+  }
+//
+// // 3. Write a function that takes a character (i.e. a string of length 1) and returns true if it is a vowel, false otherwise.
+function isVowel(character) {
+  var vowels = ['a','e','i', 'o', 'u']
 
-// Define a function maxOfThree() that takes three numbers as arguments and returns the largest of them.
+  if (vowels.indexOf(character) !== -1){
+    return true;
+  } else {
+    return false;
+  }
+};
 
-// Write a function that takes a character (i.e. a string of length 1) and returns true if it is a vowel, false otherwise.
 
-// Write a function translate() that will translate a text into "rövarspråket".
+
+// 4. Write a function translate() that will translate a text into "rövarspråket".
 // That is, double every consonant and place an occurrence of "o" in between. For example, translate("this is fun") should return the string "tothohisos isos fofunon".
 
-// Define a function sum() and a function multiply() that sums and multiplies (respectively) all the numbers in an array of numbers.
+function translate(text) {
+  var vowels = ['a','e','i', 'o', 'u', ' ']
+  // string
+  // turn the string into an array
+  var textArray = text.split('')
+
+  // check for consonant
+  for (i=0; i < textArray.length; i++) {
+
+    if (vowels.indexOf(textArray[i]) === -1) {
+    //'consonant' + 'o' + 'consonant'
+
+    textArray[i] = textArray[i] + 'o' + textArray[i]
+
+    }
+  }
+  // turn back to a string
+  return translatedText = textArray.join('');
+
+}
+
+
+
+// 5. Define a function sum() and a function multiply() that sums and multiplies (respectively) all the numbers in an array of numbers.
 // For example, sum([1,2,3,4]) should return 10, and multiply([1,2,3,4]) should return 24.
 
+  function sum(numbers) {
+    var total = 0;
+    for (var i = 0; i < numbers.length; i++) {
+      total = total + numbers[i];
+    }
+    return total;
+  };
+
+
+  function multiply(numbers) {
+
+      var total = 1;
+      for (var i = 0; i < numbers.length; i++) {
+          total = total * numbers[i];
+      }
+      return total;
+  }
```

Comment:
Good solutions. Javascript also provides very useful functions to achieve these objectives:
```
+function sum( arr ) {
 +  return arr.reduce( ( accumulator, value ) => accumulator + value, 0 )
 +}
 +
 +function multiply( arr ) {
 +  return arr.reduce( ( accumulator, value ) => accumulator * value )
 +}